modules/verifying-serving-with-openshift-ingress-sharding-configuration.adoc

```diff
@@ -0,0 +1,38 @@
+:_content-type: PROCEDURE
+[id="verifying-serving-with-openshift-ingress-sharding-configuration_{context}"]
+= Verifying Serving with OpenShift ingress sharding configuration
+
+After configuring ingress sharding, Knative Serving, and your service, you can verify that your service uses the correct route and the selected ingress shard.
+
+.Procedure
+
+. Print information about the services in the cluster:
++
+.Example command
+[source,terminal]
+----
+$ oc get ksvc
+----
++
+.Example output
+[source,terminal]
+----
+NAME         URL                                                             LATESTCREATED      LATESTREADY        READY   REASON
+hello-dev    https://hello-dev-default.dev.serverless.cluster.example.com    hello-dev-00001    hello-dev-00001    True
+hello-prod   https://hello-prod-default.prod.serverless.cluster.example.com  hello-prod-00001   hello-prod-00001   True
+----
+
+. Verify that your service uses the correct route and the selected ingress shard:
```

Comment:
Not sure if this is an example or a literal command. If an example:
```suggestion
. Verify that your service uses the correct route and the selected ingress shard by running a command similar to the following:
```

If it's the literal command, just the normal "by running the following command"